mongo/integration/unified_spec_test.go

```diff
@@ -36,13 +36,23 @@ import (
 )
 
 const (
-	gridFSFiles       = "fs.files"
-	gridFSChunks      = "fs.chunks"
-	cseMaxVersionTest = "operation fails with maxWireVersion < 8"
+	gridFSFiles  = "fs.files"
+	gridFSChunks = "fs.chunks"
 )
 
 var (
 	defaultHeartbeatInterval = 50 * time.Millisecond
+	skippedTestDescriptions  = map[string]struct{}{
```

Comment:
Nit: instead of mapping to an empty struct, can this map to a string representing the reason? Perhaps something like the following:

```
skippedTestDescriptions  = map[string]string {
    "operation fails with maxWireVersion < 8": "SPEC-1403: This test checks to see...",
    "Ignore network timeout error on find": "GODRIVER-2123: The two tests below use a failpoint..."
}
```

That reason could be used in the call to `mt.Skip` to make it clearer when reading test logs why a test was skipped.